src/modules/tasks.js

```diff
@@ -0,0 +1,101 @@
+// import renderTodos from '..';
+import { todoInput } from './do.js';
+
+// Vars
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+let editTodoId = -1;
+const doList = document.querySelector('.do-list');
+
+export default class Tasks {
+  constructor(indexNum, description, completed) {
+    this.indexNum = indexNum;
+    this.description = description;
+    this.completed = completed;
+  }
+
+    static saveTodo = () => {
+      const todoValue = todoInput.value;
+
+      // check if the todolist is empty
+      const isEmpty = todoValue === '';
+
+      if (isEmpty) {
+        alert('To do list is empty');
+      } else if (todos.some((todo) => todo.description.toUpperCase() === todoValue.toUpperCase())) {
+        alert('Todo already exists!');
+      } else {
+        if (editTodoId >= 0) {
+          // update the edit todo
+          todos = todos.map((todo, index) => ({
+            ...todo,
+            description: index === editTodoId ? todoValue : todo.description,
+          }));
+
```

Comment:
- Kindly make sure that when you add a new task the local storage is updated. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/199986052-0f33dad0-5ae9-406d-98ff-03af8ee0ce37.png)

- Also please check that the index of the tasks starts at 1 not at 0 as the[ last code reviewer suggested](https://github.com/The-Normal-One1/To-Do-List/pull/2#discussion_r1013385462). :+1: :smiley: :smiley: